docs/reference/google.auth.credentials_async.rst

```diff
@@ -1,7 +0,0 @@
-google.auth.credentials\_async module
```

Comment:
@crwilcox Should we keep the RSTs for the async files or is it alright to delete them for now?

Reply:
Not sure I understand the 'for now' here? I would assume we still want them?

Reply:
The way `docgen` is set up it seems to try to wipe these. Opened https://github.com/googleapis/google-auth-library-python/issues/685 to track.